src/components/FolderView.css

```
@@ -0,0 +1,21 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0 32px;
+}
+
+.FolderView {
```

요 클래스 네이밍도 케밥케이스로 변경되면 좋을 것 같아요
```suggestion
.folder-view {
```